### DISCUSSION TITLE
[Windows] How to convert PCM 16bit to IEEEFloat?

### DISCUSSION THREAD

**Discussion by lostmsu** [naudio/NAudio#1118] [Q&A]
I am trying to create `WaveFormatConversionProvider`, but with 
- from: 16 bit PCM: 48000Hz 2 channels
- to: 32 bit IEEFloat: 48000Hz 2 channels

the constructor fails with `AcmNotPossible calling acmStreamOpen`

**Comment by markheath** [comment]
`WaveFormatConversionProvider` tries to use ACM codecs, but you should be able to use `ToSampleProvider` extension method instead which can do this conversion without needing to use ACM.
